src/System/executable_query.rs

```diff
@@ -57,6 +66,13 @@ fn not_satisfied(query_str: &str) {
     println!("{} -- Property is NOT satisfied", query_str);
 }
 
+fn print_path(path: Vec<Transition>) {
```

Comment:
You can't just take a reference?
It also is very sus to me that you are printing this.

Reply:
Is used in main on line 61, where the print_result is used with is a implenentation on the QueryResult enum.
Have fixed so it is a reference